### DISCUSSION TITLE
"Error" window get stretched if window border is relatively big

### DISCUSSION THREAD

**Discussion by Oxore** [davatorium/rofi#1431] [Q&A] [answered]
After number of experiments I narrowed down my `~/.config/rofi/config.rasi` to this:
```css
window {
    border: 5% 0;
}
```

I run `rofi` like this:
```sh
rofi -modi run -show run -no-default-config
```
I get what I expect on the "run" window (full screen shot):

![run](https://user-images.githubusercontent.com/17534278/132128529-75c0bd54-3b4b-41f0-8f73-c94c4c5583f5.png)

As you can see I type in `asdf` command, which does not exist in my system and then I get error message in, as I understand, a special "error" window:

![error](https://user-images.githubusercontent.com/17534278/132128572-21d27281-de2c-4472-835b-5cdcca67cce7.png)

There are two things appear unexpected to me:
1. The "error" window is stretched all the way down, I can't even tell where it ends.
2. Top edge of the "error" window is right at the center of the "run" window.

If I set window border to `4% 0` like this:
```css
window {
    border: 4% 0;
}
```

I get this result:

![error2](https://user-images.githubusercontent.com/17534278/132128760-dd30fe42-8ea7-4f88-b8eb-3cbcf925b583.png)

On this screenshot behavior 1 is gone, but behavior 2 is still present.

The same happens when I use `px` instead of `%` for border size. Pixel value must be around `50px` or above to reproduce the stretching problem in my case.

Is there a way to configure "error" window to make it **centered** and **not get stretched** when relatively large window border value is set?

**Comment by DaveDavenport** [comment, marked as answer]
Interesting, I wonder what is going wrong here. 
Ill upgrade this to an issue, because this seems to be unwanted behavior.
I had not seen this before.

**Reply by DaveDavenport** [reply]
#1437

**Reply by Oxore** [reply]
Thank you. I will be watching the Issue. As far as I understand all further discussion will be conducted in the issue. I think I should mark this discussion as answered.

**Reply by Oxore** [reply]
Or should I? I don't understand the discussion-issue workflow of Rofi project.

**Reply by Oxore** [reply]
My actual use case is now solved by using fullscreen + padding configuration instead of border - see https://github.com/davatorium/rofi/issues/1437#issuecomment-914151059.

At least one of two behaviors described here is a bug and fix is about to be merged right now. See #1437. I am marking this discussion as answered.

Thank you, @DaveDavenport!
